### DISCUSSION TITLE
LayoutTo appears to be not working?

### DISCUSSION THREAD

**Discussion by noobsoftware** [dotnet/maui#11253] [Q&A] [answered]
When calling LayoutTo on any StackLayout i see no animation take place, has LayoutTo been changed somehow or is it broken?

I've also noticed another thing that seems to be a bug. When trying to read a JS file from the Raw resources folder it doesn't work however when changing the filename extension to .txt it works fine. Not sure if this is a bug either.

**Comment by vallgrenerik** [comment]
I'm also hitting this one. Did you find any solutions?

**Comment by filipnavara** [comment, marked as answer]
It's filed as an issue: https://github.com/dotnet/maui/issues/15439
